src/main/java/com/google/step/servlets/RestaurantPostListServlet.java

```diff
@@ -40,10 +37,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     Restaurant restaurant = manager.createRestaurantWithBlobKey(name, photoBlobkey);
-    List<Deal> deals = new ArrayList<>();
-    List<String> placeIds = Arrays.asList(places.split(","));
-    response.getWriter().println(JsonFormatter.getRestaurantJson(restaurant, deals, placeIds));
```

Comment:
Just checking - this is never used on the frontend?

Reply:
yes, because it redirects to the restaurant page, so the json isn't used